src/main/java/org/moera/node/text/markdown/mention/internal/MentionsNodeRenderer.java

```diff
@@ -42,7 +42,7 @@ private void render(MentionNode node, NodeRendererContext context, HtmlWriter ht
                 try {
                     RegisteredNameDetails details = options.namingCache.get(name);
                     name = details.getNodeName() != null ? details.getNodeName() : name;
-                } catch (NamingNotAvailableException e) {
+                } catch (NamingNotAvailableException|NullPointerException ignore) {
```

Comment:
Catching `NullPointerException` is too radical. If you know that `options.namingCache` may be `null`, just check this.